pyvista/plotting/renderer.py

```diff
@@ -2743,7 +2741,7 @@ def enable_depth_of_field(self, automatic_focal_distance=True):
         See :ref:`depth_of_field_example` for a full example using this method.
 
         """
-        self._render_passes.enable_depth_of_field_pass()
+        self._render_passes.enable_depth_of_field_pass(automatic_focal_distance)
```

Comment:
Sneaky bugfix.

Reply:
Wouldn't be an @akaszynski PR without one...